src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -125,9 +132,9 @@ public static object GetVariableValue(Document document, ISchema schema, Variabl
                 throw;
             }
 
-            if (input == null && variable.DefaultValue != null)
+            if (input == null)
             {
-                return variable.DefaultValue.Value;
+                return null;
             }
```

Comment:
This change represents a semantic change from 3.2 to 3.3.  Specifically, GetVariableValue now returns `null` for variables set to null rather than the variable's default value.

Reply:
```c#
/// <summary>
/// Return the specified variable's value for the document from the attached <see cref="Inputs"/> object.
/// Since v3.3 returns null for variables set to null rather than the variable's default value.
/// </summary>

Let's document it as is.
```

Reply:
Done